.eslintrc

```diff
@@ -20,7 +20,7 @@
   "rules": {
     "eol-last": ["error", "always"],
     "indent": ["warn", 2],
-    "max-len": ["error", { "code": 95, "ignoreComments": true }],
```

Comment:
Why ?

Reply:
To write longer code line like [this one](https://github.com/johannchopin/gitmoji-browser-extension/pull/74/files#diff-52b3e2692dcfb066a0d97b2585d3e6e7f1674ac744449e8e1e75709d5cb044b2R44)

Reply:
:white_check_mark: Valid example :+1: